src/is-mobile.ts

```diff
@@ -0,0 +1,3 @@
+const MOBILE_MAX_WIDTH = 1024;
```

Comment:
1024 não é muito?, tipo, se for menor que 1024 é mobile. Mas pegando a doc do Primeflex por exemplo, o `small` é considerado abaixo de 576.